modules/monitoring-running-metrics-queries.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="running-metrics-queries_{context}"]
+= Running metrics queries
+
+You begin working with metrics by entering one or several Prometheus Query Language (PromQL) queries.
+
+.Procedure
+
+. Open the OpenShift Container Platform web console and navigate to *Monitoring* -> *Metrics*.
+
+. In the query field, enter your PromQL query.
+* To show all available metrics and PromQL functions, use the "Insert Metric at Cursor" button.
```

Comment:
s/use the "Insert Metric at Cursor" button/click **Insert Metric at Cursor**/

Reply:
Implemented.